go.mod

```diff
@@ -2,6 +2,8 @@ module kpt.dev/configsync
 
 go 1.23.0
 
+replace github.com/GoogleContainerTools/kpt => ./internal/third_party/github.com/GoogleContainerTools/kpt
```

Comment:
Suggest adding a comment here to clarify that this is only temporary and we plan to remove the dependency later

Reply:
Will do in my next PR, since this already merged